go/libkb/context.go

```diff
@@ -690,3 +692,11 @@ func (m MetaContext) Keyring() (ret *SKBKeyringFile, err error) {
 	}
 	return m.ActiveDevice().Keyring(m)
 }
+
+var _ logger.CtxAndLogger = MetaContext{}
+
+func (m MetaContext) UpdateContext(c context.Context) logger.CtxAndLogger {
```

Comment:
`MetaContext` already has something like this --- `WithContext`.  It's important that these things are always passed by value since `context.Context` always need to be passed by value.

Reply:
Rename to something like.... `UpdateContextToLoggerContext` ?

Reply:
I think we can just use the WithContext method in this case. I do not need to return the interface. I'll delete this for now and we can revisit once we write the client code.

Reply:
`UpdateContextToLoggerContexter`